apps/verticalhome/js/clock.js

```diff
@@ -70,6 +61,10 @@ function Clock() {
   this.handleEvent = function cl_handleEvents(e){
 
     switch(e.type) {
+      /**
+       * This event is thrown when time format has been changed.
```

Comment:
I personally prefer "received" too, but I have also seen "thrown" used with events in some cases. However I'm not sure whether it is gramatically correct or not. :) 

Let's keep it as is here.